client/src/styles/brand.scss

```diff
@@ -7,6 +7,16 @@
   &__link:hover {
     color: var(--color-page-text);
     text-decoration: none;
+    display: inline-block;
```

Comment:
What purpose does this property serve? I tried removing it in my testing, and it didn't seem to have an impact - but perhaps it serves an edgecase I wasn't replicating?

Reply:
Hmm, you're right! Doesn't seem to have an effect, and I can't actually remember what it was for. It was possibly to solve alignment issues before I decided to move the `app-brand__logo` class from a wrapper `div` to the `img` tag itself.

Reply:
Sweet, makes sense - would you mind dropping it out? Once that's pushed we're good to merge 👍

Reply:
Done!